epytope/EpitopePrediction/PSSM.py

```diff
@@ -637,16 +636,15 @@ def __load_allele_model(allele, length):
                         score = 0.0001
                     elif score > 1e6:
                         score = 1e6
-                    result[alleles_string[a]][pep_seqs[p]] = score
-                    # print a, score, result
+                    scores[alleles_string[a]][pep_seqs[p]] = score
 
-        if not result:
+        if not scores:
             raise ValueError("No predictions could be made with " + self.name + " for given input. Check your"
                                                                                 "epitope length and HLA allele combination.")
+        
+        result = {allele: {self.name+"_score":(list(scores.values())[j])} for j, allele in enumerate(alleles)}
```

Comment:
Do we need the `self.name` part here? The method should be part of the result anyway right?

Reply:
True! I thought since (at least) syfpeithi has a different scoring system than other predictors it makes it a unique score. But that information is in the result anyway yes. I can change that to only "Score" again if you want to

Reply:
Yes please. :) I think it would be better to be consistent here.